Build system does not support HTTPS calls to Maven

**Describe the bug**
When I try to run `./refine build`, I get a 501 error when the build system attempts to contact `http://repo.maven.apache.org/`.  This appears to be related to Maven's HTTPS switchover on 2020-01-15 (https://blog.sonatype.com/central-repository-moving-to-https), which causes `http://` calls to be rejected.

**To Reproduce**
Steps to reproduce the behavior:
1. Clone the master repository
2. `./refine build`

**Current Results**
```[INFO] ------------------------------------------------------------------------
[INFO] Building OpenRefine 3.3-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Downloading: http://repo.maven.apache.org/maven2/org/codehaus/mojo/build-helper-maven-plugin/1.8/build-helper-maven-plugin-1.8.pom
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] OpenRefine ......................................... FAILURE [  3.014 s]
[INFO] OpenRefine - main .................................. SKIPPED
[INFO] OpenRefine - server ................................ SKIPPED
[INFO] OpenRefine - extensions ............................ SKIPPED
[INFO] OpenRefine - Jython extension ...................... SKIPPED
[INFO] OpenRefine - Wikidata extension .................... SKIPPED
[INFO] OpenRefine - Database extension .................... SKIPPED
[INFO] OpenRefine - Gdata extension ....................... SKIPPED
[INFO] OpenRefine - PC-axis extension ..................... SKIPPED
[INFO] OpenRefine - Phonetic clustering extension ......... SKIPPED
[INFO] OpenRefine - packaging ............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 3.927 s
[INFO] Finished at: 2020-01-15T12:02:00-05:00
[INFO] Final Memory: 7M/37M
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin org.codehaus.mojo:build-helper-maven-plugin:1.8 or one of its dependencies could not be resolved: Failed to read artifact descriptor for org.codehaus.mojo:build-helper-maven-plugin:jar:1.8: Could not transfer artifact org.codehaus.mojo:build-helper-maven-plugin:pom:1.8 from/to central (http://repo.maven.apache.org/maven2): Failed to transfer file: http://repo.maven.apache.org/maven2/org/codehaus/mojo/build-helper-maven-plugin/1.8/build-helper-maven-plugin-1.8.pom. Return code is: 501 , ReasonPhrase:HTTPS Required. -> [Help 1]
```

**Expected behavior**
The build system is able to successfully contact the Maven repository and download necessary files.

**Desktop (please complete the following information):**
 - OS: macOS 10.14.6
 - Browser Version: n/a
 - JRE or JDK Version: openjdk 12.0.1 2019-04-16

**OpenRefine (please complete the following information):**
 - Version: 3.3-SNAPSHOT

--- a/refine
+++ b/refine
@@ -304,8 +304,8 @@ tools_prepare() {
 mvn_prepare() {
     tools_prepare
 
-    MVN_VERSION="3.2.2"
-    MVN_URL="https://archive.apache.org/dist/maven/binaries/apache-maven-${MVN_VERSION}-bin.tar.gz"
+    MVN_VERSION="3.6.3"
+    MVN_URL="https://www-us.apache.org/dist/maven/maven-3/${MVN_VERSION}/binaries/apache-maven-${MVN_VERSION}-bin.tar.gz"
     MVN_FILE=`echo $MVN_URL | sed 's|.*/||'`
     MVN_DIR="apache-maven-${MVN_VERSION}"
 

